Authenticator/Source/Root.swift

```diff
@@ -175,7 +170,7 @@ extension Root {
     }
 
     @warn_unused_result
-    private func handleTokenScannerEffect(effect: TokenScannerViewController.Effect) -> Effect? {
+    private mutating func handleTokenScannerEffect(effect: TokenScannerViewController.Effect) -> Effect? {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 106 characters (line_length)